lib/GADS/Datum/Rag.pm

```diff
@@ -66,6 +68,19 @@ sub write_value
     $self->write_cache('ragval');
 }
 
+sub as_grade
+{
+    my $self = shift;
+    my %mapping = (
```

Comment:
I'd rather define this mapping in the application config and set it as `state` var at init